impeller/geometry/color.h

```diff
@@ -840,10 +840,14 @@ struct Color {
 
   static Color Random() {
     return {
+        // This method is not used for cryptographic purposes.
+        // NOLINTBEGIN(clang-analyzer-security.insecureAPI.rand)
```

Comment:
Now that's a lint I didn't expect.

Reply:
Yeah it makes sense most of the time, just not in this case!